index.html

```diff
@@ -843,11 +846,30 @@
 			*****************************************************
 		*/
 
+		// generalised code to add district info
+		function expandDistrictInfo(district) {
+			// make sure we have a district to use
+			if (district.length > 0 && district[0].layer.id === 'school_house_senate_districts_UNION-poly') {
+				data = district[0].properties;
+				var html = "<hr />";
```

Comment:
The `<hr />` as separator between program-specific info and district info is kind of ugly, I know.  But I think it's worth having something that clearly separates the two kinds of metadata.  Your call on whether to:
1. Ignore that thought and just make it a `<br />`
2. Style the `<hr />` in the CSS to make it look better
3. Make a different separator (even just two `<br />`s might do it, to be honest)

Reply:
I will take a look and likely will do whatever is fastest :)